.github/ISSUE_TEMPLATE.md

```diff
@@ -17,8 +17,20 @@ New Feature Request / Improvement:
 1. Should let user see such and such
 
 Bug:
-- Please provide steps to reproduce including browser version
+- Use 'Bug' label
+- Please provide:
+1. Summary
+1. OS/Browser, version/device used if mobile
+1. Environment url(s)
```

Comment:
Presumably you mean stage vs prod?

Reply:
right, stage vs prod.

Reply:
Cool.  I'm not sure if that will be immediately clear / obvious that will be but we can always  see how people fill it out.